Library/Homebrew/cmd/outdated.rb

```diff
@@ -123,6 +124,13 @@ def print_outdated(formulae_or_casks)
         end
       end
 
+      def print_disabled(casks)
+        casks.each do |cask|
+          opoo "#{cask.token} is #{DeprecateDisable.message(cask)}"
+        end
+        puts "Consider looking for alternatives, as they will be removed from Homebrew soon."
```

Comment:
Need to handle this (and probably also the `opoo` above differently when STDOUT is not a TTY e.g. compare `brew outdated` and `brew outdated | cat` output.